test/sample_test.yaml

```diff
@@ -63,29 +66,46 @@ spec:
                 - name: test-name
                   value: "{{item}}"
             withItems:
-              - xgboost_training_cm
               - lightweight_component
               - dsl_static_type_checking
               - pipeline_transformers
               - secret
               - sidecar
-              - dataflow
               - execution_order
               - imagepullsecrets
               - retry
               - artifact_location
               - preemptible_tpu_gpu
               - volume_snapshot_ops
-              - kubeflow_tf_serving
-              - container_build
               - loop_output
               - loop_parameter
               - loop_static
               - resource_ops
               - multiple_outputs
+            when: "{{inputs.parameters.is-integration-test}} == false"
+          - name: run-integration-tests-loop
```

Comment:
nit: integration test and e2e test reads similar, I'm not sure people can easily understand how they are different.
If you cannot find a good name, we can leave the decision to later too, not blocking

Reply:
Replied in the main thread.